app/Http/Controllers/PageController.php

```diff
@@ -28,15 +28,17 @@ public function __construct(ConfigFunctions $configFunctions)
 
 	function page(Request $request, $page)
 	{
+		Configs::get();
```

Comment:
This is a bit awkward, is there a way to make `Configs::get_value` check if the cache is not set and call `Configs::get` from within?

Reply:
it is supposed to be the case but for some reason it is not working:
https://github.com/LycheeOrg/Lychee-Laravel/blob/refactoring/app/Configs.php#L66